pkgs/tools/misc/coreutils/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, lib, buildPackages
 , autoreconfHook, texinfo, fetchurl, perl, xz, libiconv, gmp ? null
+, openssl ? null
```

Comment:
You shouldn't need the default. openssl will always be defined in a callPackage.